src/training/calculatorreview/Main.java

```diff
@@ -4,9 +4,79 @@
 
 public class Main {
 
+    private static Scanner in;
+
     public static void main(String[] args) {
+        printWelcomeMessage();
+
+        in = new Scanner(System.in);
+
+        while (true) {
+            performOneCalculation();
+        }
+    }
+
+    private static void performOneCalculation() {
+        String op = askForOperator();
+        int[] n = askForNumberArray(op);

```

Comment:
these variable names are a bit confusing , operation, number , answer would be better